dd-sdk-android-gradle-plugin/src/test/kotlin/com/datadog/gradle/plugin/DdAndroidGradlePluginFunctionalTest.kt

```diff
@@ -1025,6 +1025,118 @@ internal class DdAndroidGradlePluginFunctionalTest {
         assertThat(result).hasNoNdkSymbolUploadTasks()
     }
 
+    @Test
+    fun `M try to upload with custom ndk W custom filepath provided`(forge: Forge) {
```

Comment:
not necessary additional symbols files are built using NDK, it can be some static lib built with non-Android tooling.

```suggestion
    fun `M try to upload with additional symbol files W custom filepath provided`(forge: Forge) {
```